bin/inventory-parser.bash

```diff
@@ -0,0 +1,390 @@
+#!/bin/bash
+
+set -e -o pipefail
+
+here="$(dirname "$(readlink -f "$0")")"
+# shellcheck source=bin/common.bash
+source "${here}/common.bash"
+
+# Cache hosts info
+output_hosts_info(){
+  local filename="$1"
+  # shellcheck disable=SC2016
+  md5=$(echo -n "${filename}"| xargs -n1 -I{} bash -c 'echo "$1"; cat "$1"' x {} | md5sum | awk '{print $1}')
```

Comment:
I'm getting warnings form xargs for this line, what does it actually do?

Reply:
What kind of warning? as I didn't get any on my local machine. 
Regarding that line, the idea at first was getting the md5 of the inventory, but it seems like for dynamic ones it doesn't work, as we will be getting the same md5 for both sc/wc inventories, so I was trying to find a way to include both content + filename in the resulting md5

Reply:
That `-I` and `-n` are mutually exclusive so `-n` is skipped.

For dynamic inventories you can include `terraform.tfstate` in the checksum if it exists.

Reply:
Sure, I'll change it to get the checksum of the terraform.tfstate instead, that should be straightforward